eslint.config.mjs

```diff
@@ -54,7 +54,7 @@ const rules = {
 	'no-alert': 'error',
 	'no-array-constructor': 'off',
 	'no-async-promise-executor': 'error',
-	'no-await-in-loop': 'error',
+	'no-await-in-loop': 'off',
```

Comment:
Question: no objection but what is the rational here?

Reply:
This is an annoying eslint rule that blocks this pattern:

```typescript
for (const item of list) {
  await somethingAsync(item);
}
```

It forces us to instead do this:

```typescript
for await (const item of list) {
  await somethingAsync(item);
}
```

But this is wrong as a `for await..of` is for async generators and not async code blocks on the loop.

The rule won't cause any issues for us.